package com.fiap.g10.g10auth.exception;

public class DadoNaoEncontradoException extends RuntimeException{

    public DadoNaoEncontradoException(String mensagem) {
        super(mensagem);
    }
}
